R/checker.R

```diff
@@ -4,11 +4,21 @@ check_resolved_modules <- function(resolved_module_list) {
     log_warn(msg)
   }
 
+  if (!all(is.character(resolved_module_list[["module_id_list"]]))) {
+    msg <- "module_list has at least one module_ids that is not a character"
```

Comment:
```suggestion
    msg <- "module_list has at least one module_id that is not of type character"
```
There are some tests that check for string equality of errors, but I think this change does not impact any.